lib/oban/testing.ex

```diff
@@ -211,11 +213,11 @@ defmodule Oban.Testing do
   @spec perform_job(
           worker :: Worker.t(),
           args :: term(),
-          opts :: [Job.option() | {:repo, module()}]
+          opts :: [Job.option() | {:repo, module()} | {:testing, testing_mode()}]
```

Comment:
Would you want this documented?

Usage at the call site feels a little bit clunky and I wasn't sure if I should provide a naming alias for `:testing` for clarity purposes:

```elixir
      assert :ok = perform_job(SomeWorker, some_args, testing: :manual)
```

Reply:
Hmm. There isn't a real job to operate on in the db, so the `perform_job` helper should always be inline and skip any acking/db interaction.

I thought that this would look more like:

```elixir
  engine = Process.get(:oban_engine)

  try do
    Process.put(:oban_engine, :inline)

    # other code
  after
    Process.put(:oban_engine, engine)
  end
```